### ISSUE TITLE
map rendered somwehat strangely

### ISSUE THREAD

**Issue by anaderi** [dundalek/atom-markdown-mindmap#13]
(see on the right)
![image](https://cloud.githubusercontent.com/assets/1198881/19421625/2ba74418-940e-11e6-9bb6-16472442922c.png)

Atom: 1.13.0-dev-11f8d9e
Platform: Mac OS X (El capitan)
v0.2.4

**Comment by lyndametref** [comment]
I have the same problem. Was there an atom update that broke this package?

**Comment by luddek** [comment]
This is because of a deprecation of how markmap is getting the width and height in chrome: https://www.chromestatus.com/features/5724912467574784
I've added a pull request to markmap: https://github.com/dundalek/markmap/pull/7
If you want to change it yourself it is just two lines of code.

**Comment by lyndametref** [comment]
@luddek : Awesome, it fixed it perfectly for me!

**Comment by chenzeyu** [comment]
@luddek How can I apply your changes manually in Atom?

**Comment by luddek** [comment]
@chenzeyu 
Change line 129 and 130 in
`~/.atom/packages/markdown-mindmap/node_modules/markmap/view.mindmap.js`
to
```
    state.height = svg.node().getBoundingClientRect().height;
    state.width = svg.node().getBoundingClientRect().width;
```

**Comment by RickTorresJr** [comment]
@luddek
Required a restart of Atom but worked perfectly. Big thanks.

**Comment by ChrisChinchilla** [comment]
@dundalek How can we get this fixed? Is there a way to override in your code?

**Comment by dundalek** [comment]
This is now fixed and published, update to the latest version v0.2.5 inside Atom.
